src/DDTrace/version.php

```diff
@@ -1,5 +0,0 @@
-<?php
-
-// Must begin with a number for Debian packaging requirements
-// Must use single-quotes for packaging script to work
```

Comment:
We should preserve these comments on `DDTrace\Tracer::version()`, I think.

Reply:
Good idea, and I think that repetition helps here. Let me add all over the places where we are defining versions.